homeassistant/components/sensor/tibber.py

```diff
@@ -28,11 +28,6 @@
 async def async_setup_platform(hass, config, async_add_entities,
                                discovery_info=None):
     """Set up the Tibber sensor."""
-    if discovery_info is None:
```

Comment:
Don't remove the guard clause. Just remove the logging.

Reply:
That doesn't use config flow (`async_setup_entry`), yes.

Reply:
A mistake in the config yaml file, with an section for the platform, will otherwise try to set up the platform.